Libraries/rocSPARSE/level_2/bsrmv/README.md

```diff
@@ -137,7 +144,7 @@ $$
 
 Therefore, the BSR representation of $A$, using column-major ordering, is:
 
-```
+```math
```

Comment:
This should not be denoted with `math` as most Markdown renderers will parse this a $\LaTeX$ or KaTeX block.

```suggestion
```cpp
```

There are more of these cases, but I'm only pointing out this one to not clog up the rest of the review.